src/lib/features/project/project-service.e2e.test.ts

```diff
@@ -82,7 +82,9 @@ beforeAll(async () => {
     const config = createTestConfig({
         getLogger,
         experimental: {
-            flags: {},
+            flags: {
+                cleanApiTokenWhenOrphaned: true,
```

Comment:
That's fun. Killswitch being disabled by default was masking that tests related to other flag don't pass.